tools/etcd-dump-db/meta.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "unsafe"
+
+const magic uint32 = 0xED0CDAED
+
+type inBucket struct {
+	root     uint64 // page id of the bucket's root-level page
+	sequence uint64 // monotonically incrementing, used by NextSequence()
+}
+
+type Meta struct {
```

Comment:
Just an idea but in future could we pull some of these values from `bbolt` rather than duplicating them here? i.e. https://github.com/etcd-io/bbolt/blob/0cecda66e016f212c9d13aa74cfd93cb07948ff6/internal/common/meta.go#L12

Also https://github.com/etcd-io/bbolt/blob/0cecda66e016f212c9d13aa74cfd93cb07948ff6/internal/common/types.go#L16

Reply:
I thought about it, but I intentionally did not do it because,
- to avoid circuarly dependency
- the Meta/Page are included in internal package, so they can't be used by other repo